### PR TITLE
Fixed lint problems

### PR DESCRIPTION
**Pull request by YuriyChaban**

### CODE REVIEW

File: homeworks/yuriy.chaban_YuriyChaban/homework_2/hw1.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+rl.question('Please, indicate the height of the pyramid (number of lines) as a number greater than 0:\n', (answer) => {
+    const totalNumberOfRows = parseInt(Math.ceil(answer), 10);
+    const isAnswerIsNumber = typeof totalNumberOfRows === 'number' && !Number.isNaN(totalNumberOfRows);
+
+    if (isAnswerIsNumber && totalNumberOfRows > 0) {
```
Comment (donVladimir) [inline homeworks/yuriy.chaban_YuriyChaban/homework_2/hw1.js]: IMHO, you can reverse validation of input and exit from a function if something wrong. Therefore your main business code will be but splitted from the validation part.

Comment (donVladimir) [reply homeworks/yuriy.chaban_YuriyChaban/homework_2/hw1.js]: (answer) => {
    if (!valid1) {
        console.error('Some error 1');
        return;
    }

    if (!valid2) {
        console.error('Some error 2');
        return;
    }

    businessLogic();
}

Comment (YuriyChaban) [reply homeworks/yuriy.chaban_YuriyChaban/homework_2/hw1.js]: I can divide the conditions of the calculation! 
Need I do this?

File: homeworks/yuriy.chaban_YuriyChaban/homework_2/hw2.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+const rangesArray = [];
+let numberItem;
+let reversedNumber;
+let rightDigit;
+let minRange;
+let maxRange;
+let isMinIsNumber;
+let isMaxIsNumber;
+let isMaxAndMinNumber;
```
Comment (donVladimir) [inline homeworks/yuriy.chaban_YuriyChaban/homework_2/hw2.js]: The way how you implement this is very similar to using global variables, and it tends to complex errors. On another hand, you could use a more functional approach, when functions are pure - get value, calculate and return the result.

Comment (donVladimir) [reply homeworks/yuriy.chaban_YuriyChaban/homework_2/hw2.js]:
```
let r;

// function with side effect
function sum(a, b) {
     r = a + b;
}

// pure function
function sum(a, b) {
     return a + b;
}
```

pure functions much easier to test, and they more predictable.

**Pull request merged**